### PR TITLE
fix(curriculum): replace repl.it with replit.com

### LINKED ISSUE #41444: Repl Migration
Just putting this here to keep on radar:

Replit are moving from `repl.it` to `replit.com`.
- https://blog.repl.it/dotcom

They say:
> Nothing is required on your end -- all urls will automatically redirect.

---

The whole platform needs to be audited for the change from `repl.it` to `Replit`, as per:
- https://github.com/freeCodeCamp/freeCodeCamp/pull/41933/files#r621498674

### PR DESCRIPTION
**Pull request by GemmaAprendiendo**
Replace repl.it with replit.com in the English version only.  Chinese and Spanish versions have the same issue.  First PR I do so please let me know if anything is wrong.  I only checked under the curriculum folder.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to #41444

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md
Comment (ShaunSHamilton) [inline curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md:14]:
```suggestion
-   Use [our Replit starter project](https://replit.com/github/freeCodeCamp/boilerplate-project-exercisetracker) to complete your project.
```

**Review by raisedadead — APPROVED** [review]
Was able to skim through the changes, thanks to standard patterns in the curriculum.

LGTM

**Review by ShaunSHamilton — COMMENTED** [review]
Hello there,

Thank you, for taking the time to contribute.

I think all of the mentions of `repl.it` should become `Replit`, as this was the purpose of Replit changing their name - `.com` is not part of the brand, but `it` always was.

I have not made this a change request, but just a comment, so as to allow others to comment on whether this is just completely irrelevant.

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md
Comment (raisedadead) [reply curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md:14]: That's a good catch. 

However, I see that the original issue does not mention this explicitly. It may make sense to audit all places instead of only the curriculum (as in this PR) and make the change.

Comment (nhcarrigan) [reply curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md:14]: I agree that this can be merged as is and an additional PR could be created to change the name

**Comment by gmfuster** [comment]
@ShaunSHamilton Oh I see it now, I should have let the part of the text as only Replit and have the .com only on the actual links.  Should I make that change and send another PR?  I'm not sure how to cancel this one or how that works.  Sorry about that.

**Comment by raisedadead** [comment]
You should wait for this PR to be merged and create a fresh PR afterwards. When you do, you may want to audit other places in the codebase and not just the curriculum.

**Comment by gmfuster** [comment]
@raisedadead cool, I'll do that then.  After this one is merged, I'll go back to these files and remove the .com from the text (not the link).  And just to be sure, when you say not just the curriculum you mean everything under the freeCodeCamp folder itself?  I'm new to this so trying to understand.  Thanks again for your patience.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Comment by raisedadead** [comment]
First,

> I'm new to this so trying to understand. Thanks again for your patience.

Ask as many questions you like; we are here to help you help us.

And just to be sure, when you say not just the curriculum you mean everything under the freeCodeCamp folder itself? 

Yes - that is correct. You can skip the translations i.e: `curriculum/challenges/espanol` & `curriculum/challenges/chinese` etc. We get these automatically once the English version gets updated.

**Comment by ShaunSHamilton** [comment]
@gmfuster I will update the issue with what still needs to be done.

Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

---

To let you know: If you want to make changes to a PR, you do not need to close it - just make the changes on the same branch, and push that new commit to the branch; the PR will be updated.

**Comment by raisedadead** [comment]
Ignore me, listen to Shaun, he is smarter and will guide you with the changes. 😉

**Pull request merged**
